sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -1349,6 +1482,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ConditionalAccessPolicyErrorHelpLink
+
+A Link for help when Conditional Access Policy blocks a user.
```

Comment:
We can add a comment that it should be a valid URL and a example. I have tried and if you put any string you will get "WarningConditionalAccessPolicyHelpLinkInvalidUrl".

Reply:
done, thanks